coralproject/cay
# Restoring friendly field names

## What does this PR do?
- Fixes #589 
  (restores field names on drop)
## How do I test this PR?
- Go to **Create form**
- Drag a "Long Answer" field to the list
- Its name should be "Long Answer" and not "TextArea". All fields should have friendly names.

@coralproject/frontend
